src/golang/lib/models/shared/object.go

```diff
@@ -0,0 +1,9 @@
+package shared
+
+type Object string
```

Comment:
same thing about moving this to the `notification.go` combined file
and rename to `NotificationObject`